### PR TITLE
Correct a small makefile issue

### PR DESCRIPTION
**Pull request by geeknoid**
- Was passing a incorrect directory when building the protoc-gen-doc plugin. Not sure how come this worked before,
but the directory was definitely incrrect.

- Sync to the latest doc gen tool.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -235,7 +235,11 @@ $(mixer_template_pb_gos) $(mixer_template_pb_doc) : $(mixer_template_protos) | d
 mixer/v1/config/fixed_cfg.pb.go mixer/v1/config/istio.mixer.v1.config.pb.html: mixer/v1/config/cfg.proto | depend $(protoc_gen_gogo) $(protoc_bin)
 	# Generate mixer/v1/config/fixed_cfg.pb.go (requires alternate plugin and sed scripting due to issues with google.protobuf.Struct)
 	@$(protoc) $(proto_path) $(gogo_plugin) $(protoc_gen_docs_plugin)mixer/v1/config $^
-	@sed -e 's/*google_protobuf.Struct/interface{}/g' -e 's/ValueType_VALUE_TYPE_UNSPECIFIED/VALUE_TYPE_UNSPECIFIED/g' mixer/v1/config/cfg.pb.go >mixer/v1/config/fixed_cfg.pb.go
+	@sed -e 's/*google_protobuf.Struct/interface{}/g' \
+	    -e 's/ValueType_VALUE_TYPE_UNSPECIFIED/VALUE_TYPE_UNSPECIFIED/g' \
+	    -e 's/package config/package istio_mixer_v1_config/g' \
```
Comment (douglas-reid) [inline Makefile]: i don't think we want to do this. let's fix the package in istio/istio

**Comment by geeknoid** [comment]
Can we deal with the naming in a separate PR? I'm trying to get a bunch of doc work done, and the fact this is currently broken makes it so I can't do "go generate" on the protos in istio/istio. With this fix, the end to end pipeline is working.

**Comment by douglas-reid** [comment]
Let's use this: https://github.com/istio/istio/pull/3146.

The package in istio/istio is not needed. And we will simplify the codegen.

**Comment by douglas-reid** [comment]
@geeknoid we'll also need something like: https://github.com/istio/api/pull/356.

**Comment by geeknoid** [comment]
PTAL

**Pull request merged**